src/main/java/com/erii/core/TaskManager.java

```diff
@@ -0,0 +1,191 @@
+package com.erii.core;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskManager {
+
+    public enum Priority {
+        SS, S, A, B, C, D
+    }
+
+    public abstract class Task {
```

Comment:
avoid declaring multiple classes within a same file, try an separate them into differen files (to improve readability)